### DISCUSSION TITLE
Difference between Invitatorium1 and ...2 etc.

### DISCUSSION THREAD

**Discussion by themortenbay** [DivinumOfficium/divinum-officium#2986] [Q&A] [answered]
Hi.

Still working my way through the psalm translations for my Danish version (the Bible translation I'm working with turned out to be sub-optimal).

I noticed in Psalterium there are four invitatories (Invitatorium1, Invitatorium 2, Invitatorium 3, Invitatorium 4).

My question is, what is the difference between these four translations? I assume one of them is used in the daily invitatory and another for the Epiphany?

Also, it would be interesting to know what English translations were chosen to represent he different renditions of the Latin.

**Comment by gregordick** [comment, marked as answer]
* `Invitatorium.txt`: This is the basic case.
* `Invitatorium1.txt`: This one is in fact not an invitatory at all, but is the special arrangement of Psalm 94 for Matins of the Epiphany in the third nocturn.
* `Invitatorium2.txt`: Sundays in Septuagesimatide: Omits "præoccupémus fáciem ejus in confessióne, et in psalmis jubilémus ei", because of the similarity to the antiphon.
* `Invitatorium3.txt`: Passiontide: Omits "Hódie, si vocem ejus audiéritis, nolíte obduráre corda vestra", because of the similarity to the antiphon, and also omits the Gloria Patri.
* `Invitatorium4.txt`: Mondays: Omits "Veníte, exsultémus Dómino", because this is the text of the antiphon.

With the exception of `Invitatorium1.txt`, the differences versus `Invitatorium.txt` are therefore all omissions, and translations can be adapted accordingly.  `Invitatorium1.txt` gives (most of) the text of the psalm from the Gallican psalter, so more care might be needed here.  The English translations in DO are correspondingly different, but I'm afraid I don't know the origins of each.
